packages/terra-collapsible-menu-view/src/CollapsibleMenuView.jsx

```diff
@@ -110,12 +122,7 @@ class CollapsibleMenuView extends React.Component {
       <div {...customProps} className={collapsibleMenuViewClassName} ref={this.setContainer}>
         {visibleChildren}
```

Comment:
wouldn't it make more sense to put it back in the visibleChildren array to be displayed here rather than have it display in the menu div

Reply:
...yeah probably

Reply:
So after taking a look with how the width is calculated, some logic would need to be added to determine if the ellipse menu button should show or not. Given this, I think this will be a cleaner implementation.